components/eventing-controller/pkg/backend/jetstreamv2/utils.go

```diff
@@ -47,7 +47,6 @@ func (js *JetStream) getDefaultSubscriptionOptions(consumer SubscriptionSubjectI
 		toJetStreamConsumerDeliverPolicyOptOrDefault(js.Config.JSConsumerDeliverPolicy),
 		nats.MaxAckPending(maxInFlightMessages),
 		nats.MaxDeliver(jsConsumerMaxRedeliver),
-		nats.AckWait(jsConsumerAcKWait),
```

Comment:
Should we not keep `ackWait` in case the call `msg.NakWithDelay(jsConsumerNakDelay)` fails?
if the nack function call fails for any reason, it will not be redelivered anymore.

Reply:
done